code-studio/src/js/initApp/loadApp.js

```diff
@@ -51,7 +51,12 @@ module.exports = function (callback) {
       appOptions.disableSocialShare = true;
     }
 
-    $.ajax('/api/user_progress/' + appOptions.scriptName + '/' + appOptions.stagePosition + '/' + appOptions.levelPosition).done(function (data) {
+    $.ajax(
+        '/api/user_progress/' + appOptions.scriptName +
+        '/' + appOptions.stagePosition +
+        '/' + appOptions.levelPosition +
+        '/' + appOptions.serverLevelId
```

Comment:
We can now do this with [template strings](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)!